resources/views/components/post/post-header-meta.blade.php

```diff
@@ -0,0 +1,8 @@
+<div class="flex flex-wrap items-center">
+    @foreach ($post['tags'] as $tag)
+        <x-tag-hash :tag="$tag"></x-tag-hash>
+    @endforeach
+    <span class="ml-2 lg:ml-10">
+        {{ $post['published_at']->formatLocalized('%d %B %Y') }}
```

Comment:
quick fix to pass the tests: 

```suggestion
        {{ $post->is_published ? $post->published_at->formatLocalized('%d %B %Y') : __('post.not_published') }}
```

(now they are failing, because you are trying to format `published_at` date - even when it might be `null` for not-published posts